docs/manual/optional-checker.tex

```diff
@@ -98,6 +98,36 @@
 \label{fig-optional-hierarchy}
 \end{figure}
 
+\subsectionAndLabel{Optional method annotations}{optional-method-annotations}
+
+The Optional Checker supports several annotations that specify method
+behavior.  These are declaration annotations, not type annotations:  they
+apply to the method itself rather than to some particular type.
+
+\begin{description}
+
+\item[\refqualclass{checker/optional/qual}{RequiresPresent}]
+  indicates a method precondition:  The annotated method expects the
+  specified variables to be a present Optional when this
+  method is invoked.  Don't use this for formal parameters (just annotate
+  their type as \<@Present>).  \<@RequiresPresent> is appropriate for
```

Comment:
Does this last sentence mean that `@RequiresPresent` is useful methods that require a`@MaybePresent` field to be present? If so, can you reword the sentence.